Runtime/RosMsgs/Vision/vision_msgs.cs

```diff
@@ -0,0 +1,102 @@
+using std_msgs;
+using std_msgs.msg;
+using geometry_msgs.msg;
+using sensor_msgs.msg;
+
+namespace vision_msgs
+{
+    namespace msg
+    {
+        public class BoundingBox3D : IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.BoundingBox3D";
+            }
+
+            /// <summary>
+            /// The 3D position and orientation of the bounding box center.
+            /// </summary>
+            public Pose center;
+
+            /// <summary>
+            /// The size of the bounding box, in meters, surrounding the object's center pose.
+            /// </summary>
+            public Vector3 size;
+        }
+
+        public class ObjectHypothesisWithPose : IRosMsg
+        {
+            string IRosMsg.GetRosType()
+            {
+                return "vision_msgs.msg.ObjectHypothesisWithPose";
+            }
+
+            /// <summary>
+            /// The unique numeric ID of object detected.
+            /// </summary>
+            public long id;
+
+            /// <summary>
+            /// The probability or confidence value of the detected object. By convention,
+            /// this value should lie in the range [0-1].
+            /// </summary>
+            public double score;
+
+            /// <summary>
+            /// The 6D pose of the object hypothesis. This pose should be
+            /// defined as the pose of some fixed reference point on the object, such a
+            /// the geometric center of the bounding box or the center of mass of the
+            /// object.
+            /// </summary>
+            public PoseWithCovariance pose;
+
+        }

```

Comment:
Need to add ROS2 class.

make sure that id and score these two variables are inheritors of the class [ObjectHypothesis](https://github.com/ros-perception/vision_msgs/blob/ros2/vision_msgs/msg/ObjectHypothesis.msg)